.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+stages:
+  - build
+  - publish
+
+variables:
+  NODE_VERSION: "20"
+
+.node_template:
+  image: node:${NODE_VERSION}
+  cache:
+    key: ${CI_COMMIT_REF_SLUG}
+    paths:
+      - .npm/
+
+build:
+  stage: build
+  extends: .node_template
+  script:
+    - npm ci --cache .npm --prefer-offline
+    - npm run test
```

Comment:
Do we have tests to run?

Reply:
Yes, in Both liblab v1 and v2 we include a test script in the package.json